lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -21,14 +21,16 @@ def self.github_token!
 
       def self.github_client
         @@client ||= begin
-          Octokit::Client.new(access_token: github_token!)
+          client = Octokit::Client.new(access_token: github_token!)
```

Comment:
Turns out if we don't define this, it's hard to pass it out of the block 🤦

Reply:
🤦‍♂️ Sorry for misleading you on that one with my review on #313, reading the diff on GitHub I thought the client variable that I made you remove was out of the begin/end block, not inside of it… oops.

Nice touch adding a unit test for that 👍